### DISCUSSION TITLE
Unexpected argument error when using "-m" during testing

### DISCUSSION THREAD

**Discussion by k-g-j** [Cyfrin/foundry-full-course-cu#52] [Q&A]
I am getting the error "error: unexpected argument '-m' found" when running ```forge test -m testFunction``` is "-m" not valid and you have to use "--match-test" ?

**Comment by PatrickAlphaC** [comment]
Yep! They recently deprecated `-m` for `--mt`.

You can see it here: https://github.com/Cyfrin/foundry-full-course-f23/blob/main/chronological-updates.md#lesson-7

**Reply by k-g-j** [reply]
Thank you!

**Reply by AnouarBF** [reply]
Thank you that helped !

**Comment by albin-george-kurian** [comment]
use -mt instead of -m

correct code:
```
forge test -mt FUNCTION_NAME
```
